qchem/tests/hf/test_integrals.py

```diff
@@ -818,7 +818,7 @@ def test_gradient_repulsion(symbols, geometry, alpha, coeff):
 
 @pytest.mark.parametrize(
     ("n", "t", "f_ref"),
-    [(1.25, 0.00, 0.2857142857142857), (2.75, 1.23, 0.061750771828252976)],
+    [(1.25, np.array([0.0]), 0.2857142857142857), (2.75, np.array([1.23]), 0.061750771828252976)],
```

Comment:
I don't know why exactly it was required to cast the exponent as an array of floats. Maybe add a test case in which `t` is provided multiple entries, if it makes sense to do this (It might be the case that the only time we call the private function `_boys` is on `t` values with only a single entry in the array)

Reply:
Used array of multiple number for `t`.